WordPress/src/main/java/org/wordpress/android/ui/posts/PrepublishingBottomSheetFragment.kt

```diff
@@ -186,6 +188,6 @@ class PrepublishingBottomSheetFragment : WPBottomSheetDialogFragment(),
 
     override fun onPublishButtonClicked() {
         viewModel.onCloseClicked()
-        prepublishingBottomSheetListener?.onPublishButtonClicked()
+        arguments?.getBoolean(PUBLISH_POST)?.let { prepublishingBottomSheetListener?.onPublishButtonClicked(it) }
```

Comment:
I'm not 100% sure of this approach. One of the reasons, this PR is still marked as draft so I can think about it. I was wondering if I should propagate this value all the way down to the publish button. 

I was also wondering if this should be stored in the `EditPostActivity` I didn't want to put anything else in it since it's such a God class. I will think about this some more but if you do look at this, you can share your thoughts.